pkgs/build-support/php/builders/default.nix

```diff
@@ -0,0 +1,8 @@
+{ callPackage, ... }:
+{
+  v1 = {
+    buildComposerProject = callPackage ./v1/build-composer-project.nix { };
+    mkComposerRepository = callPackage ./v1/build-composer-repository.nix { };
+    composerHooks = callPackage ./v1/hooks { };
```

Comment:
`callPackage` should import a file that contains a function that returns a single derivation.

Reply:
Thanks, I just fixed this one.